src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -81,12 +82,20 @@ const ArticlePageMostReadSection = styled(MostReadSection)`
 
 const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
   const { articleAuthor } = useContext(ServiceContext);
+  const { state } = useLocation();
   const headline = getHeadline(pageData);
   const description = getSummary(pageData) || getHeadline(pageData);
   const firstPublished = getFirstPublished(pageData);
   const lastPublished = getLastPublished(pageData);
   const aboutTags = getAboutTags(pageData);
 
+  useEffect(() => {
+    if (state?.anchor) {
```

Comment:
I'm not sure what this does? Does this scroll to the element with this ID?

If there is a hash in the url then the browser scrolls to that by default.

Reply:
Thanks for the review @jroebu14. The browser only scrolls on initial load. Unfortunately we have to manually simulate this behaviour when the page is CSR. This is known issue ReactTraining/react-router#394. Most react developer use [react-router-hash-link](https://www.npmjs.com/package/react-router-hash-link) as a workaround but that's too heavy for Simorgh.